tools/run_python.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# Copyright 2011 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+#
+# Entpy point for running python scripts on UNIX systems.
+
+if [ -z $PYTHON ]; then
+  PYTHON=$(which python3)
+fi
+
+if [ -z $PYTHON ]; then
```

Comment:
Note that it's possible for Python to be installed in a path with spaces, which breaks this line: #11005